examples/umbrella_sampling/umbrella.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import numpy as np
+
+import hoomd
+hoomd.context.initialize()
+import hoomd.md as md
+import hoomd.dlext
+
+import pysages
+from pysages.ssages.collective_variables import Component
+from pysages.ssages.methods import UmbrellaSampling
+from pysages.backends import bind
+
+def main():
+    with hoomd.context.SimulationContext() as context:
+        system = hoomd.init.read_gsd("start.gsd")
+
+        #gather basic lognames
+        qr = []
+        #log some thermo properties
+        qr += ['temperature', 'potential_energy', 'kinetic_energy']
+
+        cvs = (Component([0], 0), Component([0], 1))
+
+        method = UmbrellaSampling(cvs, 10., (-2, 2))
+
+        hoomd.md.integrate.nvt(group=hoomd.group.all(), kT=1, tau=0.1)
+        hoomd.md.integrate.mode_standard(dt=0.05)
+
+        bind(context, method)
```

Comment:
```suggestion
        sampler = bind(context, method)
```

To do manual logging for tests purposes, you might want to read `sampler.state.xi` after a number of time steps. Not really necessary for the PR though.

Reply:
well, with the merged #45 I decided to purposefully not assign the result of bind in order to test the global sampler pool.